### DISCUSSION TITLE
What should a block of code be called: chunk or cell?

### DISCUSSION THREAD

**Discussion by mine-cetinkaya-rundel** [quarto-dev/quarto-cli#6151] [Q&A]
Quarto documentation refers to both "code chunks" and "code cells". Quarto internals generally refer to code cells. It would be good if we could settle on a term and use it, but this might be impossible. I still wanted to record this discussion here, in case others have thoughts and want to chime in.

Some points to keep in mind:

- R Markdown and knitr refer to them as "chunks"
- Jupyter refers to them as "cells" (https://jupyterbook.org/)
- Pluto.jl also refers to them as "cells" (https://plutojl.org/)

**Comment by mcanouil** [comment]
Personally, I use "code cell(s)" and would promote this at the risk of little "confusion".
My current experience is that the transition from "code chunks" to "code cells" seems smooth at least in my current job where my colleagues are more familiar with RMarkdown.

**Reply by mcanouil** [reply]
Few statistics:
- In <https://github.com/quarto-dev/quarto-web>:
  - "code chunk" is used 94 times in 36 files.
  - "code cell" is used 269 times in 90 files.
- In <https://github.com/quarto-dev/quarto-cli>:
  - "code chunk" is used 61 times in 18 files.
  - "code cell" is used 138 times in 41 files.

**Reply by mcanouil** [reply]
It seems Quarto is mostly using "code cell" rather than "code chunk".
Some consistency is required so, I'll open an issue to work on this.

**Comment by mcanouil** [comment]
Closing this in favour of #6220 to make the naming consistent in CLI and documentation.
